### ISSUE TITLE
Verbum failing to install at .msi installation (ref. #123)

### ISSUE THREAD

**Issue by CWilsonTSP** [ferion11/LogosLinuxInstaller#126]
Made a new issue since it wont let me reopen the last issue.
Previous issue was a bad URL structure. Below is the failed installation after the download.

Below is the final comment on issue #123

---

              > I just merged a fix for this. I tested it and it should be working. Please reopen if there are more problems.

It seems there is now an issue at installation, with the installation hanging after the download:
```
Installing VerbumBible 64bits…
Verbum_vInstaller-x64.msi exists. Using it…
Running: /usr/bin/wine64 msiexec /i /home/myusername/VerbumBible10/data/Verbum_vInstaller-x64.msi
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
00d0:fixme:file:NtLockFile I/O completion on lock not implemented yet
00d0:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
00d8:fixme:imm:ImeSetActiveContext (0000000000010026, 0): stub
00d8:fixme:imm:ImmReleaseContext (0000000000010020, 0000000000010026): stub
00d0:fixme:imm:ImeSetActiveContext (0000000000010056, 1): stub
00d0:fixme:imm:ImmReleaseContext (000000000001006A, 0000000000010056): stub
```

_Originally posted by @CWilsonTSP in https://github.com/ferion11/LogosLinuxInstaller/issues/123#issuecomment-1397156912_

**Comment by thw26** [comment]
Ahh, it appears the rename of the file didn't work:

`/home/myusername/VerbumBible10/data/Verbum_vInstaller-x64.msi`

Let me see if there is some error in the logic.

**Comment by thw26** [comment]
Okay. I resolved this in #127. Apologize the last one did not resolve it. I tested the installer and Verbum installs and the login page launches. Let me know if you encounter any other problems.

**Comment by CWilsonTSP** [comment]
could `002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
` be a big issue? I'm still having trouble, but now this is the only error I see (and it was on this one too)
